tests/e2e/utils/flinkStatement.ts

```diff
@@ -87,31 +86,13 @@ export async function stopStatement(webview: FrameLocator) {
   await stopButton.click();
 
   // Wait for the statement to be stopped
-  await expect(webview.getByTestId(FlinkStatementTestIds.statementStatus)).toHaveText("STOPPED");
+  await expect(webview.getByTestId("statement-status")).toHaveText("STOPPED");
 
   // Verify the statement detail info
-  await expect(webview.getByTestId(FlinkStatementTestIds.statementDetailInfo)).toHaveText(
+  await expect(webview.getByTestId("statement-detail-info")).toHaveText(
     "This statement was stopped manually.",
   );
 
   // Verify no error is shown
-  await expect(webview.getByTestId(FlinkStatementTestIds.statementDetailError)).toBeHidden();
-}
-
-export async function verifyStatementStatus(webview: FrameLocator, status: string) {
-  await expect(webview.getByTestId(FlinkStatementTestIds.statementStatus)).toHaveText(status, {
-    // If the statement was just submitted, it may take a while to transition to the new status.
-    timeout: 30_000,
-  });
-}
-
-/**
- * Assert that the result stats are correct. This is the text that appears at the bottom of the Results Viewer tab,
- * e.g. "Showing 1..100 of 200 results (total: 200).".
- *
- * @param webview - The webview page object.
- * @param stats - The expected result stats.
- */
-export async function verifyResultsStats(webview: FrameLocator, stats: string) {
-  await expect(webview.getByTestId(FlinkStatementTestIds.resultsStats)).toHaveText(stats);
```

Comment:
These were only used in the Flink statement tests in one spot, and eslint was complaining about no assertions in the tests themselves. Best for all parties to just remove these functions and put the assertions in the tests.